pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -58,10 +63,12 @@ type ClusterInfoOptions struct {
 func NewCmdClusterInfo(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
 	o := &ClusterInfoOptions{
 		IOStreams: ioStreams,
+		Color:     true,
 	}
 
 	cmd := &cobra.Command{
-		Use:     "cluster-info",
+		Use: "cluster-info",
+		DisableFlagsInUseLine: true,
```

Comment:
Why you need this?

Reply:
I have no reason. I saw this widely used in other commands. I can remove this if we don't want this.